server/src/main/java/org/elasticsearch/action/search/SearchRequest.java

```diff
@@ -305,6 +309,20 @@ public SearchRequest requestCache(Boolean requestCache) {
     public Boolean requestCache() {
         return this.requestCache;
     }
+    
+    /**
+     * Sets if this request should allow partial results. By default (not set, or null,
+     * will default to the cluster level setting if partial results are accepted or not).
+     */
+    public SearchRequest allowPartialSearchResults(Boolean allowPartialSearchResults) {
```

Comment:
I think we should make this accept a `boolean`

Reply:
Do we not need a Boolean to understand when an explicit choice hasn't been made by the client? It's when the value is null that we substitute for the cluster-level default.

Reply:
ah - or did you know this and just mean to make the setters take a bool and keep the getters returning Boolean?

Reply:
yes I meant just the setters